fem/pbilinearform.hpp

```diff
@@ -101,6 +101,10 @@ class ParBilinearForm : public BilinearForm
        diagonal for this case. */
    void AssembleDiagonal(Vector &diag) const override;
 
+   /// Returns the matrix assembled on the true dofs, i.e. P^t A P.
+   /** The returned matrix is owned by the form. */
+   HypreParMatrix *ParallelAssembleInternal();
```

Comment:
I don't think we want method called `*Internal` to be part of the public interface. :)

It is best to either move to be `protected` or rename it.

Reply:
ping @v-dobrev ☝️